data/human/culture conversations.txt

```diff
@@ -120,6 +120,35 @@ mission "Paradise Cleaner Incident"
 
 
 
+mission "Class Trip to Port"
+	minor
+	source
+		attributes "south"
+		not attributes "station"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`A group of children wave at you as you step off your ship, marveling at the spacecraft on display as their teachers try and fail to hold their attention. You can see the fading of their uniform colors from afar as their group walks past, yet none of them look one bit displeased. While you make your way into the spaceport itself, you wonder how many of them may eventually own their own spaceship and fly across the stars.`
+				decline
+
+
+
+mission "Sojourn Sunday Sales"
+	minor
+	source
+		government "Free Worlds"
```

Comment:
This specific mission might fit better on a Syndicate world.

Reply:
It's meant to show that the affluent urban Free Worlds are still much like everywhere else.

Reply:
Fair enough, makes sense.